applications/MultilevelMonteCarloApplication/python_scripts/adaptive_refinement_utilities.py

```diff
@@ -204,3 +187,27 @@ def EstimateMeshSizeCurrentLevel(self):
                 interp_error_level = original_interp_error
             mesh_size_level = self.mesh_size_coarsest_level*np.sqrt(interp_error_level/original_interp_error) # relation from [Alauzet] eqs. pag 34 and 35
```

Comment:
```suggestion
            mesh_size_level = self.mesh_size_coarsest_level*math.sqrt(interp_error_level/original_interp_error) # relation from [Alauzet] eqs. pag 34 and 35
```
You can use math.sqrt here and avoid importing numpy

Reply:
Done in ceb00be.